src/experiment.py

```diff
@@ -150,7 +150,7 @@ def plot_column(self, ycol: int, plotdir: str = "plots"):
         plt.clf()
         plt.close()
 
-    def plot_angle_histogram(self, angle_x, angle_y, plotdir: str = "plots", numbins = 100):
+    def plot_angle_histogram(self, angle_x, angle_y, numbins, plotdir = None):
```

Comment:
This code is useful, however it violates the principle of consistent returns - it basically serves two purposes and it depends on non-null value of one of its arguments. I am going to accept it for now but we will surely need to refacteor this soon. This is also marked by `pylint` as a finding - therefore I am not going to create a separate issue.